compiler_opt/es/blackbox_optimizers.py

```diff
@@ -550,17 +561,17 @@ def run_step(self):
     self.x = x_next
     self.k += 1
 
-  def get_solution(self):
+  def get_solution(self) -> npt.NDArray:
     return self.x
 
-  def get_x_diff_norm(self):
+  def get_x_diff_norm(self) -> float:
     return self.x_diff_norm
 
-  def get_iterations(self):
+  def get_iterations(self) -> int:
     return self.k
 
 
-def make_projector(radius):
+def make_projector(radius: float) -> Callable:
```

Comment:
Callable from what to what?

Reply:
The function details have been added now